src/Database/Persist/Documentation.hs

```diff
@@ -344,6 +344,19 @@ markdownTableRenderer = Renderer{..}
    showType SqlBlob      = "blob"
    showType SqlBool      = "boolean"
    showType (SqlOther t) = t
+   
+   cleanComment :: Text -> Text
+   cleanComment comment =
+     let
+       -- Newline characters may be present in a comment. The need to be removed
+       -- to allow the markdown tables to render properly.
+       newlineToBr '\n' = "<br>"
+       newlineToBr c = Text.singleton c
+     in
+      case Text.unsnoc comment of
+        Nothing -> comment
+        Just (front, '\n') -> Text.concatMap newlineToBr front
+        Just _ -> Text.concatMap newlineToBr comment
```

Comment:
May be worthwhile to use [`Text.strip`](https://www.stackage.org/haddock/lts-18.3/text-1.2.4.1/Data-Text.html#v:strip). Leading and/or trailing whitespace won't get rendered anyway, and it should clean out the implementation nicely.

Reply:
done!